File: kizizip/01/[PGS - Lv1] 핸드폰 번호 가리기.java
class Solution {
    public String solution(String phone_number) {
        int mask = phone_number.length() - 4;
        
        if (mask > 0) {
            String answer = "*".repeat(mask) + phone_number.substring(mask);
            return answer; 
        } else return phone_number;
        
    }
}

File: JeongEon8/01/[PGS - Lv1] 핸드폰 번호 가리기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string phone_number) {
    string answer = "";
    int size = phone_number.length();
    int star_size = size - 4;
    for(int i= 0; i < phone_number.length(); i++){
        if(i < star_size){
            answer += "*";
        }
        else{
            answer += phone_number[i];
        }
    }
    return answer;
}
